lte/cloud/go/services/lte/obsidian/handlers/handlers.go

```diff
@@ -300,21 +304,41 @@ func listEnodebs(c echo.Context) error {
 	if nerr != nil {
 		return nerr
 	}
+	var pageSize uint64 = 0
+	var err error
 
-	ents, _, err := configurator.LoadAllEntitiesOfType(
+	if pageSizeParam := c.QueryParam(ParamPageSize); pageSizeParam != "" {
```

Comment:
We're probably going to be doing a lot of parsing page size and page token. Create a helper fn somewhere under orc8r/cloud/go/obsidian, or something like that, that takes a c and returns page size, page token, error

Reply:
agreed will do